bitnami/spark/templates/svc-worker-metrics.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
```

Comment:
I thought that the original service would work without the need of a new service, right?

Reply:
I move to PodMonitor, this way there is no necessity to create new services.